modules/backend/layouts/_mainmenu.htm

```diff
@@ -59,7 +59,7 @@
                 <a
                     href="javascript:;" onclick="$.oc.layout.toggleAccountMenu(this)"
                     title="<?= e(trans('backend::lang.account.signed_in_as', ['full_name' => $this->user->full_name])) ?>">
-                    <img src="<?= $this->user->getAvatarThumb(90, ['mode' => 'crop', 'extension' => 'png']) ?>" class="account-avatar" />
+                    <img role="img" loading="lazy" src="<?= $this->user->getAvatarThumb(90, ['mode' => 'crop', 'extension' => 'png']) ?>" alt="<?= e(trans($this->user->full_name)) ?>" class="account-avatar" width="78" height="78" />
```

Comment:
@ayumihamsaki why is `$this->user->full_name` passed through `trans()`? A user's name is not going to be a translation key.

Reply:
My bad, didn't know that.